hnn_core/gui/_viz_manager.py

```diff
@@ -49,26 +48,84 @@
 ]
 
 fig_templates = {
-    "2row x 1col (1:3)": {
-        "kwargs": "gridspec_kw={\"height_ratios\":[1,3]}",
+    "[Blank] 2row x 1col (1:3)": {
+        "kwargs": {
+            "gridspec_kw": {"height_ratios": [1, 3]}
+        },
         "mosaic": "00\n11",
     },
-    "2row x 1col (1:1)": {
-        "kwargs": "gridspec_kw={\"height_ratios\":[1,1]}",
+    "[Blank] 2row x 1col (1:1)": {
+        "kwargs": {
+            "gridspec_kw": {"height_ratios": [1, 1]}
+        },
         "mosaic": "00\n11",
     },
-    "1row x 2col (1:1)": {
-        "kwargs": "gridspec_kw={\"height_ratios\":[1,1]}",
+    "[Blank] 1row x 2col (1:1)": {
+        "kwargs": {
+            "gridspec_kw": {"height_ratios": [1, 1]}
+        },
         "mosaic": "01\n01",
     },
-    "single figure": {
-        "kwargs": "",
+    "[Blank] single figure": {
+        "kwargs": {
+            "gridspec_kw": ""
+        },
         "mosaic": "00\n00",
     },
-    "2row x 2col (1:1)": {
-        "kwargs": "gridspec_kw={\"height_ratios\":[1,1]}",
+    "[Blank] 2row x 2col (1:1)": {
+        "kwargs": {
+            "gridspec_kw": {"height_ratios": [1, 1]}
+        },
         "mosaic": "01\n23",
+    }
+}
+
+data_templates = {
+    "Drive-Dipole": {
```

Comment:
```suggestion
    "Drive-Dipole (2 x 1)": {
```

I kind of like knowing how many subplots I'll get ... but it's a personal preference

Reply:
I added the subplot grid spec to the layout template name